crates/uv-virtualenv/src/virtualenv.rs

```diff
@@ -53,6 +55,7 @@ pub(crate) fn create(
     prompt: Prompt,
     system_site_packages: bool,
     allow_existing: bool,
+    clear: bool,
```

Comment:
I'm using a `bool` here, but could also use an `enum` to represent three mutually exclusive policies: default (don't remove or overwrite), clear, and overwrite/allow-existing.